src/imageio/storage/piwigo.c

```diff
@@ -108,8 +116,8 @@ typedef struct dt_storage_piwigo_params_t
   char *album;
   gboolean new_album;
   int privacy;
-  gboolean export_tags; // deprecated - let here not to change params size. to be removed on next version change
```

Comment:
here we change the params (1 field removed ; one added) of the module, in this case we need:
- to bump the version `DT_MODULE(1)` -> `DT_MODULE(2)` (see top of file)
- add a new routine named `legacy_params()` see `galery.c `for inspiration (this routine is in charge from upgrading the params from version 1 to version 2, you need to ensure that the fields are all properly initialized as you have removed one in the middle - `export_tags` - it needs a bit more coding).

Note doing that will break all previous user's presets.

Reply:
Don't user presets go through legacy_params too? Or did you mean "Not doing that"?

Reply:
I meant "Not doing that..." indeed. Sorry for the typo.

Reply:
did undo of params change, as I don't think this justifies a version bump to DT_MODULE(2)